src/util/bundleUtils.js

```diff
@@ -266,9 +266,73 @@ function replaceReferences(entries) {
   return newEntries;
 }
 
+/**
+ * Checks that the parameters input to $evaluate-measure are valid. Throws a ServerError
+ * for missing parameters, the use of unsupported parameters, and the use of unsuppported
+ * report types.
+ * @param {Object} req http request object
+ */
+function validateEvalMeasureParams(req) {
```

Comment:
I think this bundleUtils.js file is getting a little crowded now. And this function has nothing to do with a bundle. Can we move this to its own file? Maybe measureOperationsUtils or something?

Reply:
good point, I agree that it should be in a new file